configs/envoy_service_to_service.template.json

```diff
@@ -156,6 +156,47 @@
         ]
       }
     }]
+  },
+  {
```

Comment:
Leave a comment saying that this listener has a dynamic route table instead of a static one.